### ISSUE TITLE
Question/Ideas about Tables and Grids

### ISSUE THREAD

**Issue by rabbers** [GetPublii/Publii#735]
I can see that tables can be added in WISYWYG editor and in the usual fashion in Markdown editor. I do not however see tables in the block editor.

It is my thought that the block editor would be amazing if had a "Table" block - In this manner, each cell of the table could have it's own collection of nested blocks. Each utilizing the same interface as you have now, albeit with a narrower width as the layout dictates.

I was also thinking that perhaps your Markdown editor could be a block in it's own right in the Block editor. This would be amazing if it rendered a preview when the block was not being actively edited (and would also give basic access to tables).

Finally, I've been looking and I don't see anything to control or set up responsive grids and controls. The use of a grid system appears to be the domain of the theme designer. It occurs to me that in the first instance, a theme designer could subvert the table HTML (by way of theme specific classes) to acheive some kind of multi column/responsive layout from the editor.

These are just ideas, and I haven't really checked out how side-bars and the like might function in Publii, but it's sometimes just nicer to be able to use a grid where you need one. What are your thoughts in this area?

Thanks.

**Comment by JOduMonT** [comment]
> These are just ideas, and I haven't really checked out how side-bars and the like might function in Publii, but it's sometimes just nicer to be able to use a grid where you need one. What are your thoughts in this area?

If I understand well, it's all about flexbox ;)
https://www.w3schools.com/css/css3_flexbox.asp

**Comment by rabbers** [comment]
@JOduMonT - Flexbox is probably how this would be implemented, but my post is more about suggesting that the editor needs to be able to support blocks that have columns within them.

**Comment by JOduMonT** [comment]
Yes it would be awesome to being able to make a grid without table
also have snippets of CSS and HTML would be great
